### ISSUE TITLE
Automatically create inexistent workspace when loggin into account

### ISSUE THREAD

**Issue by rdumont** [vtex/toolbelt#195]
When logging into an account, you are required to choose a workspace. Sometimes you want to choose a workspace that doesn't exist yet, or you are not sure if it does. In the case of it not existing, you should be prompted to create it.

## Expected Behavior
```
$ vtex login
? Account: pokestore
? Email: dumont@vtex.com
? Code: XXXXX
? Workspace: dumontnew
? Workspace dumontnew doesn't exist. Do you want to create it? Yes
info: Worksapce dumontnew created
info: Login successful!
```

## Current Behavior
```
$ vtex login
? Account: pokestore
? Email: dumont@vtex.com
? Code: XXXXX
? Workspace: dumontnew
error:  StatusCodeError (404)
```

**Comment by tamorim** [comment]
@rdumont This was already implemented, but when we changed from `request` to the `requisition` library, the errors coming from the http client changed and thats why this happens now.
I believe the correct thing would be to fix the implementation on the `@vtex/api` module.

**Comment by tamorim** [comment]
@rdumont I was wrong! The implementation on `@vtex/api` is correct. The key on the object changed the name from `Code` to `code`. It's fixed on 739bac45893d705dc18aeda18124c91fcc4c592e
I published the new version on the `next` tag in npm .
